src/services/bundleAnalysis/useBundleSummary.tsx

```diff
@@ -0,0 +1,190 @@
+import { useQuery } from '@tanstack/react-query'
+import { z } from 'zod'
+
+import { MissingHeadReportSchema } from 'services/comparison'
+import {
+  RepoNotFoundErrorSchema,
+  RepoOwnerNotActivatedErrorSchema,
+} from 'services/repo'
+import Api from 'shared/api/api'
+import type { NetworkErrorObject } from 'shared/api/helpers'
+import A from 'ui/A'
+
+const BundleDataSchema = z.object({
```

Comment:
just want to confirm, none of these params are required? Similarly with the BundleAnalysisReportSchema

Reply:
Yes they are non-null in the schema, everything was non-null by default beforehand and we're doing a much better job ensuring things aren't non-null when we're able to now.

Reply:
Got it, thanks for the clarification